src/components/Header/Header.module.css

```diff
@@ -1,21 +1,5 @@
 .headerToolbar {
     display: flex;
     align-items: center;
-    padding: 20px;
 };
 
```

Comment:
I am thinking because our mobile and tablet header layouts are the same we only need a breakpoint at 768px and 1280px really. Let me know what you think.